web/template/layout/team-selection.gotmpl

```diff
@@ -9,7 +9,7 @@
       <label class="govuk-label" for="change-team">
         Change view
       </label>
-      <select class="govuk-select ignore-pagination-bottom" label for="pagination-selected-team" name="change-team" onchange="window.location.replace('/supervision/workflow/?change-team=' + this.value)">
+      <select class="govuk-select ignore-pagination-bottom" for="pagination-selected-team" data-control="select-submit" name="change-team">
```

Comment:
I prefer using `data-module` to match the Design System, and prefixing with `app-` to distinguish from GDS/MOJ components:
```suggestion
      <select class="govuk-select ignore-pagination-bottom" for="pagination-selected-team" data-module="app-select-submit" name="change-team">
```

Reply:
Ah, good call. Having updated.